diff --git a/BAS16H.prm b/BAS16H.prm
index ee00f2b..ba2c9b3 100644
--- a/BAS16H.prm
+++ b/BAS16H.prm
@@ -1,17 +1,17 @@
 *
-. *
-.MODEL BAS16H D(
-+ IS=6.0289E-9
-+ N=2.1283
-+ RS=1.0000E-3
-+ IKF=37.638E-3
-+ CJO=593.63E-15
-+ M=.1
-+ VJ=.68774
-+ ISR=2.0646e-9
-+ NR=4.9950
-+ BV=113.60
-+ IBV=10
-+ TT=4.5000E-9)
-.ENDS
+*
+.MODEL BAS16H D(
++ IS=6.0289E-9
++ N=2.1283
++ RS=1.0000E-3
++ IKF=37.638E-3
++ CJO=593.63E-15
++ M=.1
++ VJ=.68774
++ ISR=2.0646e-9
++ NR=4.9950
++ BV=113.60
++ IBV=10
++ TT=4.5000E-9)
+*.ENDS
 *
diff --git a/BZV85-C33.prm b/BZV85-C33.prm
index 28bbb5e..61b46fa 100644
--- a/BZV85-C33.prm
+++ b/BZV85-C33.prm
@@ -1,6 +1,6 @@
 *DEVICE= BZV85-C33,D
 
-.MODEL BZV85-C33 D
+.MODEL BZV85C33 D
 + IS=110.88E-18
 + N=.92657
 + RS=.85899
@@ -12,4 +12,4 @@
 + BV=33.134
 + IBV=.37878
 + TT=2.2362E-6
-*
+*
diff --git a/PDZ4.7B.prm b/PDZ4.7B.prm
index 693b83b..70814ac 100644
--- a/PDZ4.7B.prm
+++ b/PDZ4.7B.prm
@@ -1,14 +1,14 @@
 *
-.SUBCKT PDZ4.7B 1 2
-D1 2 1 PDZ4.7B
-.MODEL PDZ4.7B D
-+ IS = 1.033E-15
-+ N = 1.001
-+ BV = 4.7
-+ IBV = 0.005
-+ RS = 0.387
-+ CJO = 2.589E-10
-+ VJ = 0.8029
-+ M = 0.3636
-+ FC = 0.5
+.SUBCKT PDZ4.7B 1 2
+D1 1 2 PDZ4.7B
+.MODEL PDZ4.7B D
++ IS = 1.033E-15
++ N = 1.001
++ BV = 4.7
++ IBV = 0.005
++ RS = 0.387
++ CJO = 2.589E-10
++ VJ = 0.8029
++ M = 0.3636
++ FC = 0.5
 .ENDS
